src/axe-extension.ts

```diff
@@ -30,4 +30,46 @@ declare module 'axe-core/axe' {
             checks: CheckConfiguration[];
         };
     };
+
+    export interface AxeRawResult {
+        id: string;
+        result: string;
+        pageLevel: boolean;
+        impact?: Impact;
+        tags: string[];
+        help: string;
+        helpUrl: string;
+        description: string;
+        inapplicable: AxeRawNodeResult[];
+        violations?: AxeRawNodeResult[];
+        passes?: AxeRawNodeResult[];
+        incomplete?: AxeRawNodeResult[];
+    }
+
+    export interface AxeRawNodeResult {
+        any: AxeRawCheck[];
+        all: AxeRawCheck[];
+        none: AxeRawCheck[];
+        node: AxeRawNode;
+        impact?: Impact;
+        result?: 'passed' | 'failed' | 'inapplicable' | 'incomplete';
+    }
+
+    export interface AxeRawCheck {
```

Comment:
It looks like `axe.d.ts` uses `CheckResult` for the v2 version of the same concept; we should probably use `AxeRawCheckResult` instead of `AxeRawCheck` for consistency